mlir/include/mlir/Dialect/Transform/LoopExtension/LoopExtensionOps.td

```diff
@@ -0,0 +1,76 @@
+//===- LoopExtensionOps.td - Transform dialect operations --*- tablegen -*-===//
```

Comment:
Some SCF transforms are currently using the `transform.loop.` prefix, e.g., https://github.com/llvm/llvm-project/blob/main/mlir/include/mlir/Dialect/SCF/TransformOps/SCFTransformOps.td#L43. Could we add a test that mixing these is fine?  (Longer-term, we should rename the SCF ones.)